components/org.wso2.carbon.identity.application.authenticator.hypr/src/test/java/org/wso2/carbon/identity/application/authenticator/hypr/HyprAuthenticatorTest.java

```diff
@@ -159,6 +160,8 @@ public void testCanHandle() {
 
         when(httpServletRequest.getParameter(HyprAuthenticatorConstants.HYPR.SESSION_DATA_KEY))
                 .thenReturn(sessionDataKey);
+        when(httpServletRequest.getParameter(HyprAuthenticatorConstants.HYPR.AUTH_TYPE))
+                .thenReturn(authType);
         Assert.assertTrue(hyprAuthenticator.canHandle(httpServletRequest));
 
```

Comment:
Add the test case related to the scenario where `authType` is null and `sessionDataKey` is a valid value.

```
        when(httpServletRequest.getParameter(HyprAuthenticatorConstants.HYPR.SESSION_DATA_KEY))
                .thenReturn(sessionDataKey);
        when(httpServletRequest.getParameter(HyprAuthenticatorConstants.HYPR.AUTH_TYPE))
                .thenReturn(null);
        Assert.assertFalse(hyprAuthenticator.canHandle(httpServletRequest));

```